openfisca_us/variables/demographic/person.py

```diff
@@ -45,6 +45,50 @@ def formula(person, period, parameters):
         )
 
 
+class is_ccdf_home_based(Variable):
+    value_type = bool
+    default_value = False
+    entity = Person
+    label = u"Whether CCDF care is home-based versus center-based"
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        return (
+            person("provider_type_group", period) != ProviderTypeGroup.DCC_SACC
+        )
+
+
+class CCDFAgeGroup(Enum):
+    I = "Infant"
+    T = "Toddler"
+    PS = "Preschooler"
+    SA = "School age"
+
+
+class ccdf_age_group(Variable):
+    value_type = Enum
+    possible_values = CCDFAgeGroup
+    default_value = CCDFAgeGroup.I
+    entity = Person
+    label = u"CCDF age group"
+    definition_period = YEAR
+
+    reference = "https://ocfs.ny.gov/main/policies/external/ocfs_2019/LCM/19-OCFS-LCM-23.pdf"
+
+    def formula(person, period, parameters):
+        age = person("age", period)
+        home_based = person("is_ccdf_home_based", period, parameters)
+        return select(
+            [
+                (age < 1.5 & ~home_based) | (age < 2 & home_based),
```

Comment:
```suggestion
                ((age < 1.5) & ~home_based) | (age < 2 & home_based),
```

I think the `&` operator has precedence over `<`, so this was evaluated as `age < (1.5 & (~home_based))` (which breaks).